config/config_test.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+import (
```

Comment:
We could test the `GetConfig` function as-well and more precisely the 2 use-cases on error:
![image](https://user-images.githubusercontent.com/12232750/108335997-3de39180-71dc-11eb-99f5-d87732f61f1d.png)